.travis.yml

```diff
@@ -22,7 +22,9 @@ matrix:
     - otp_release: 20.3
       elixir: 1.7.4
     - otp_release: 21.3
-      elixir: 1.8.1
+      elixir: 1.8.2
     - otp_release: 22.0
-      elixir: 1.9.1
+      elixir: 1.9.4
+    - otp_release: 22.3
```

Comment:
22.3 is not yet available on travis

Reply:
Yeah apparently not. Updated to 22.2 for now.